launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light_node_container.launch.py

```diff
@@ -44,6 +44,12 @@ def add_launch_arg(name: str, default_value=None, description=None):
     add_launch_arg(
         "output/traffic_signals", "/perception/traffic_light_recognition/traffic_signals"
     )
+    add_launch_arg(
+        "output/traffic_signals_car", "/perception/traffic_light_recognition/traffic_signals_car"
+    )
+    add_launch_arg(
+        "output/traffic_signals_ped", "/perception/traffic_light_recognition/traffic_signals_ped"
+    )
```

Comment:
```suggestion
    add_launch_arg(
        "output/traffic_signals_car", "/perception/traffic_light_recognition/car/traffic_signals"
    )
    add_launch_arg(
        "output/traffic_signals_ped", "/perception/traffic_light_recognition/pedestrian/traffic_signals"
    )
```
The topic name of traffic_signals ends with signals, we should follow it.
And it's hard to understand what `ped` means.

Reply:
fixed in
https://github.com/autowarefoundation/autoware.universe/commit/2f237e8a644b8b7ce191b74196f7a900e027bb80